qtmodern/windows.py

```diff
@@ -31,6 +31,16 @@ def mousePressEvent(self, event):
 
     def mouseMoveEvent(self, event):
         if self._mousePressed:
+            if self._window.windowState() == Qt.WindowMaximized:
```

Comment:
```suggestion
            if self._window.windowState() == Qt.WindowMaximized and PLATFORM == "Windows":
```

Reply:
Awesome change! But while this is the expected behavior on Windows it is not on OSX (not sure about Linux), it would be awesome if you could add a little clause to make it Windows only (at least until someone points out the behavior on Linux).

Reply:
The behavior looks the same on Linux than on Windows, tested with KDE.

Reply:
Sweet!

So let's change that too and only exclude in on macOS with `PLATFORM != "Darwin"`